CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
```

Comment:
`chore` is usually used for internal changes that does not affect library usage. It should be a fix in this case